typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -176,6 +177,21 @@ export class HyperlaneIsmFactory extends HyperlaneApp<ProxyFactoryFactories> {
           [IsmType.PAUSABLE]: contract,
         });
         break;
+      case IsmType.TRUSTED_RELAYER:
+        if (!this.deployer) {
+          throw new Error(`HyperlaneDeployer must be set to deploy ${ismType}`);
+        } else if (!mailbox) {
+          throw new Error(
+            `Mailbox address is required for deploying ${ismType}`,
+          );
+        }
```

Comment:
NIt: Could be `assert(this.deployer, `HyperlaneDeployer must be set to deploy ${ismType})`.
See assert in our utils package